app_server.py

```diff
@@ -202,29 +139,6 @@ def load_pagination(size, offset):
             return INVALID_INPUT.detailed(_("Invalid offset: %(offset)s", offset=offset))
     return Pagination(offset, size)
 
-def load_entrypoint(entrypoint, worklist):
```

Comment:
This now happens as part of `FacetsWithEntryPoint.from_request` -- there's no need to call it explicitly.